deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -255,13 +171,17 @@ spec:
                 - name: ROOK_DISCOVER_DEVICES_INTERVAL
                   value: 60m
                 - name: ROOK_HOSTPATH_REQUIRES_PRIVILEGED
-                  value: "false"
+                  value: "true"
                 - name: ROOK_ENABLE_SELINUX_RELABELING
                   value: "true"
                 - name: ROOK_ENABLE_FSGROUP
                   value: "true"
-                - name: ROOK_DISABLE_DEVICE_HOTPLUG
+                - name: ROOK_ENABLE_FLEX_DRIVER
                   value: "false"
+                - name: ROOK_ENABLE_DISCOVERY_DAEMON
```

Comment:
Remove this, we should leave it as true.

Reply:
Why? The discovery daemon is used to discover new devices. But as we'll be using PVs with StorageClassDeviceSets, rook doesn't need to discover any devices.

Reply:
We need it available for supporting baremetal installations, among others.